resources/tests.mjs

```diff
@@ -395,51 +370,26 @@ Suites.push({
             const cursor = page.querySelector(".react-stockcharts-crosshair-cursor");
             let x = 150;
             let y = 200;
-            const coords = (i) => ({ clientX: x + i * 10, clientY: y + i * 2, bubbles: true, cancelable: true });
+            const coords = (i) => ({ clientX: x + i * 10, clientY: y + i * 2 });
             for (let i = 0; i < 100; ) {
-                cursor.dispatchEvent("mousedown", coords(i), MouseEvent);
-                for (let j = 10; j--; )
-                    cursor.dispatchEvent("mousemove", coords(++i), MouseEvent);
-                cursor.dispatchEvent("mouseup", coords(i), MouseEvent);
+                let { clientX, clientY } = coords(i);
+                cursor.mouseDown(clientX, clientY);
+                for (let j = 10; j--; ) {
+                    ({ clientX, clientY } = coords(++i));
+                    cursor.mouseMove(clientX, clientY);
+                }
+                ({ clientX, clientY } = coords(++i));
+                cursor.mouseUp(clientX, clientY);
             }
         }),
         new BenchmarkTestStep("ZoomTheChart", (page) => {
             const cursor = page.querySelector(".react-stockcharts-crosshair-cursor");
-            let event = {
-                clientX: 200,
-                clientY: 200,
-                deltaMode: 0,
-                delta: -10,
-                deltaY: -10,
-                bubbles: true,
-                cancelable: true,
-            };
             for (let i = 0; i < 30; i++)
-                cursor.dispatchEvent("wheel", event, WheelEvent);
-
-            event = {
-                clientX: 650,
-                clientY: 200,
-                deltaMode: 0,
-                delta: 10,
-                deltaY: 10,
-                bubbles: true,
-                cancelable: true,
-            };
+                cursor.wheel(-10, -10, 200, 200);
             for (let i = 0; i < 10; i++)
-                cursor.dispatchEvent("wheel", event, WheelEvent);
-
-            event = {
-                clientX: 200,
-                clientY: 200,
-                deltaMode: 0,
-                delta: -10,
-                deltaY: -10,
-                bubbles: true,
-                cancelable: true,
-            };
+                cursor.wheel(10, 10, 650, 200);
```

Comment:
Did you mean to call scroll? I don't think there is wheel method defined anywhere.